### PR TITLE
Add Azure console access via script

### PR DESCRIPTION
**Pull request by vassyz**
### Context

I've been using the Azure console to connect to out servers and while documenting it I decided to add a script that mostly helps with managing different connection details. As far as I'm aware this is not sensitive data.

### Guidance to review

- read the markdown documentation 
- try to run the script

### Link to Trello card

https://trello.com/c/gGev4xlE/1170-console-access-on-our-azure-servers

### Checklist

- [x] Attach to Trello card
- [x] Rebased main
- [x] Cleaned commit history
- [x] Tested by running locally

### CODE REVIEW

File: bin/az-console [hunk 1]
```diff
@@ -0,0 +1,83 @@
+#!/bin/bash
+set -e
+
+ENVIRONMENT_NAME=$1
+PR_ID=$2
+
+case $ENVIRONMENT_NAME in
+  "production")
+    SUBSCRIPTION_ID="9c2dcb72-4bba-4acc-8d5f-8b8389a68238"
```
Comment (NickGraham101) [inline bin/az-console:9]: We can use the subscription name here, slightly easier to read `s165-teachingqualificationsservice-production`

File: bin/az-console [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    RESOURCE_GROUP="s165p01-rsm-pd-rg"
+    NAME="s165p01-rsm-production-wkr-cg"
+    ;;
+  "preprod")
+    SUBSCRIPTION_ID="250ff557-239a-4d89-83a4-07bdfc6218c7"
```
Comment (NickGraham101) [inline bin/az-console:14]: Subscription name is `s165-teachingqualificationsservice-test`

File: bin/az-console [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+    RESOURCE_GROUP="s165t01-rsm-pp-rg"
+    NAME="s165t01-rsm-preprod-wkr-cg"
+    ;;
+  "test")
+    SUBSCRIPTION_ID="250ff557-239a-4d89-83a4-07bdfc6218c7"
+    RESOURCE_GROUP="s165t01-rsm-ts-rg"
+    NAME="s165t01-rsm-test-wkr-cg"
+    ;;
+  "dev")
+    SUBSCRIPTION_ID="9816b7b0-58ca-4972-b25b-c7dbd3ee1c6d"
```
Comment (NickGraham101) [inline bin/az-console:24]: Subscription name is `s165-teachingqualificationsservice-development`

**Comment by vassyz** [comment]
[Console access on our Azure servers](https://trello.com/c/gGev4xlE/1170-console-access-on-our-azure-servers)

**Review by NickGraham101 — APPROVED** [review]

**Pull request merged**